README.md

```diff
@@ -106,6 +106,30 @@ Currently, we support the following types of assets:
 - [ERC-721](https://ethereum.org/en/developers/docs/standards/tokens/erc-721/) (Non-fungible tokens)
 - [ERC-1155](https://ethereum.org/en/developers/docs/standards/tokens/erc-1155/) (Multi-tokens)
 
+### Input encodings for deposits
+
+As explained above, when a portal teleports assets to the execution layer, it adds an input to the DApp's input box, which will then need to be interpreted and validated by the off-chain machine. In order to do that, the machine will need to understand and decode the input's payload.
```

Comment:
I would change the phrasing here a bit. It seems like the addition of an input is simply a side effect of some magical operation, which is the teleportation of assets. Rather, the teleportation is only possible with the addition of an input. So, I think it would be more accurate if we said something like... **in order** for the portal to teleport assets from the base layer to the execution layer, it adds an input to the DApp's input box, [...].